tests/tests/test_scenario.py

```diff
@@ -25,6 +58,17 @@ class _ScenarioResponse:
     train_schedules: Iterable[Any]
 
 
+@dataclass(frozen=True)
+class _ScenarioResponse:
```

Comment:
What is the difference between this and _ScenarioResponse from tests/test_scenario.py? Do you have a specific usecase for these classes?

Reply:
Would it make sense to update the other one, then?